design-proposals/vpro-eim.md

```diff
@@ -0,0 +1,185 @@
+# Design Proposal: Extensions to Edge Infrastructure Manager Core services and APIs
+
+Author(s): Edge Infrastructure Manager Team
+
+Last updated: 05/13/2025
+
+## Abstract
+
+To properly manage vPRO/Active Management Technology (AMT)/Intel Standard Manageability (ISM) services in the Edge
+Orchestrator is required to extend Infrastructure Manager APIs and
+[data model](https://github.com/open-edge-platform/infra-core/blob/main/inventory/docs/inventory-er-diagram.svg).
+
+This design proposal elaborate on the extensions to be introduced and the User flows to be supported.
+
+## Proposal
+
+Changes are expected in the data-model and a bit of re-structuring in the existing `HostResource` should be evaluated:
+
+- `BaremetalControllerKind` - identifies the type of Device Management
+- `BmcIP` - the IP of the Device Management controller
```

Comment:
As per my understanding we are not supporting BMC right now in the EMF. We are only targeting AMT/ISM. Please remove BMC related stuff from this ADR.

Reply:
Ram this is just a cleanup. There is no plan to support BMC - and honestly I would clean the mess we are doing today in OM with these fields...